supervisor/backups/manager.py

```diff
@@ -169,7 +174,7 @@ async def _load_backup(tar_file):
         tasks = [
             self.sys_create_task(_load_backup(tar_file))
             for path in self.backup_locations
-            for tar_file in _list_backup_files(path)
+            for tar_file in _list_backup_files(path, self.sys_resolution)
```

Comment:
Do we have an easy way to tell if path is a local directory or not? OSError bad message could mean something different. Looking at Linux source, it seems unlikely we'll see this on a SMBFS/NFS... But if possible we should only mark the local ones as unhealthy.